modules/proc_configuring-by-using-the-ocp-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-the-notificationsconfiguration-by-using-the-web-console_{context}"]
+= Configuring the NotificationsConfiguration CR by using the web console
+
+You can configure the `NotificationsConfiguration` custom resource by using the web console.
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges and are logged into the web console.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance. For more information, see "Enabling notifications with an Argo CD instance".
+
+.Procedure
+
+. Go to the *NotificationsConfiguration* page and click `default-notifications-configuration`.
```

Comment:
@Dhruv-Soni11 Can you please mention which perspective we must use to go to this page?

Reply:
Fixed.